### PR TITLE
Bumping to v0.5.2

### PR DESCRIPTION
**Pull request by AntoninKns**
Bumping SolverBenchmark to v0.5.2 in order to adapt to NLPModels v0.19 and latest versions of DataFrames v1.2 and v1.3.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -27,12 +27,12 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 BenchmarkProfiles = "0.2.0, 0.3.0, 0.4"
 BenchmarkTools = "^0.4.2, 0.5, 0.6, 0.7, 1"
 ColorSchemes = "^3.9"
-DataFrames = "^0.21, 1.0"
+DataFrames = "^0.21, 1.0, 1.2, 1.3"
```
Comment (abelsiqueira) [inline Project.toml:30]:
```suggestion
DataFrames = "^0.21, 1"
```
Does it break for 1.1? Otherwise, I believe using `1` is enough for all `1.x`.

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Thanks for the PR. I had a small comment.

File: Project.toml (hunk 1 above)
Comment (AntoninKns) [reply Project.toml:30]: It didn't break for 1.1 so it seems a good idea indeed

**Review by abelsiqueira — APPROVED** [review]
Thanks for the update, looks good to me.

**Pull request merged**